polygeist/tools/cgeist/Test/Verification/triple.cu

```diff
@@ -1,6 +1,9 @@
 // RUN: cgeist --use-opaque-pointers --target aarch64-unknown-linux-gnu %s %stdinclude -S -o - | FileCheck %s -check-prefix=MLIR
 // RUN: cgeist --use-opaque-pointers --target aarch64-unknown-linux-gnu %s %stdinclude -emit-llvm -S -o - | FileCheck %s -check-prefix=LLVM
 
+// FIXME: Failing possibly due to different configuration in CI
+// XFAIL: *
+
```

Comment:
Why is this failing?

Reply:
Looks like is failing with 
```
/__w/llvm/llvm/build/lib/clang/17/include/cuda_wrappers/cmath:27:15: fatal error: 'cmath' file not found
#include_next <cmath>
              ^~~~~~~
390 error(s) generated
```
in CI.

Reply:
Yes, not sure whether we want to address this now as this is not our main concern